integration/kubernetes/k8s-inotify.bats

```diff
@@ -7,6 +7,7 @@
 
 load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
+fc_limitations="https://github.com/kata-containers/documentation/issues/351"
```

Comment:
Thanks @liubin, but this variable is defined multiple times:

```bash
$ grep -r fc_limitations=
integration/kubernetes/k8s-ro-volume.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-volume.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-cpu-ns.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-projected-volume.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
integration/kubernetes/k8s-credentials-secrets.bats:fc_limitations="https://github.com/kata-containers/documentation/issues/351"
```

So instead, why don't we define it once in `./integration/kubernetes/tests_common.sh` since that is `source`'d by all the k8s bats files?

Reply:
@jodh-intel fixed as your suggestion.